src/Moq/SetupCollection.cs

```diff
@@ -82,7 +82,7 @@ public void RemoveAllPropertyAccessorSetups()
 
 			lock (this.setups)
 			{
-				this.setups.RemoveAll(x => x.Method.IsPropertyAccessor());
+				this.setups.RemoveAll(s => s is MethodSetup ms && ms.Method.IsPropertyAccessor());
```

Comment:
This is a bad change, but acceptable if it means we'll soon be able to remove this whole method.